Tools/wasm/emscripten/make_libffi.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set +e
+
+export CFLAGS="-O2 -fPIC -DWASM_BIGINT"
+export CXXFLAGS="$CFLAGS"
+
+# Build paths
+export CPATH="$PREFIX/include"
+export PKG_CONFIG_PATH="$PREFIX/lib/pkgconfig"
+export EM_PKG_CONFIG_PATH="$PKG_CONFIG_PATH"
+
+# Specific variables for cross-compilation
+export CHOST="wasm32-unknown-linux" # wasm32-unknown-emscripten
```

Comment:
Why did this end up being `wasm32-unknown-linux` instead of `wasm32-unknown-emscripten`?

Reply:
Good question. Maybe I put it that way while debugging and didn't notice before merging.